### PR TITLE
Refactor the resetting of queues and exchanges in RabbitMQContext

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: src/NServiceBus.RabbitMQ.Tests/When_sending_a_message_over_rabbitmq.cs [hunk 1]
```diff
@@ -11,6 +12,8 @@
     [TestFixture]
     class When_sending_a_message_over_rabbitmq : RabbitMqContext
     {
+        private string queueToReceiveOn = "testEndPoint";
```
Comment (bording) [inline src/NServiceBus.RabbitMQ.Tests/When_sending_a_message_over_rabbitmq.cs]: This could be const

**Comment by adamralph** [comment]
@Particular/rabbitmq-transport-maintainers anything left to do before this can be merged?

**Comment by WojcikMike** [comment]
@adamralph LGTM but I am not sure to which modifiers @bording refered to

**Comment by bording** [comment]
> @adamralph LGTM but I am not sure to which modifiers @bording refered to

I'm talking about the `private` modifiers added [here](https://github.com/Particular/NServiceBus.RabbitMQ/pull/280/files#diff-0817534c899f5c7965dec1e9cc9a1884R96)  and [here](https://github.com/Particular/NServiceBus.RabbitMQ/pull/280/files#diff-80dd104ecac36083c972b2a1ccd76715R15). Do we want to enforce the "no unneeded explicit modifiers" rule in the test code like we do in the main code base?

**Comment by adamralph** [comment]
I pushed a fixup! to remove the explicit `private` modifiers.

I guess we could enforce this for test projects but I'm not sure how to do so. That can be done in a separate PR though.

**Comment by adamralph** [comment]
@Particular/rabbitmq-transport-maintainers I pushed another fixup! to remove the remaining `private`.

**Comment by bording** [comment]
@adamralph I just noticed after merging this that this PR included `src/.nuget/v3.4.4/NuGet.exe` I'm assuming this was a mistake?

**Comment by adamralph** [comment]
Doh! Yes. Thanks for raising the fixup PR.

**Pull request merged**